test/classes/ferguson/generic-inherit6.bad

```diff
@@ -1 +1 @@
-generic-inherit6.chpl:13: syntax error: non-Identifier expression cannot be inherited
+generic-inherit6.chpl:13: syntax error: unsupported non-Identifier inherits expression
```

Comment:
I mis-parsed this as though "Inherits" was a verb rather than a type of expression.  My first reaction was to put inherits in single-quotes, but since it's not a keyword, I don't think that's a good fix.  We could hyphenate inherits-expression, but that seems a little weird.  "inheritance expression" would probably address it.  We could also call it a "parent class expression" and drop the "non-identifier"  ("unsupported parent class expression").

If "non-identifier" sticks around, let's at least decapitalize it (I realize you inherited that).

Reply:
+1 to "unsupported parent class expression" or "invalid ...."

Reply:
I'd say "cannot inherit from non-identifier".  Though even that gets fuzzy for me - I think we want to avoid compiler terms in error messages as much as possible, so would avoid expression and identifier if we can.

Reply:
Would "invalid parent class" work ok?

Reply:
> Would "invalid parent class" work ok?

How about this:

`error: invalid parent class; please specify a single class name`
or
`error: invalid parent class; only a single class name is allowed`